src/order_backup.cpp

```diff
@@ -27,17 +27,15 @@
 OrderBackupPool _order_backup_pool("BackupOrder");
 INSTANTIATE_POOL_METHODS(OrderBackup)
 
-/** Free everything that is allocated. */
 OrderBackup::~OrderBackup()
 {
-	if (CleaningPool()) return;
+}
 
-	Order *o = this->orders;
-	while (o != nullptr) {
-		Order *next = o->next;
-		delete o;
-		o = next;
-	}
+/**
+ * Create an empty order backup, filled in during SaveLoad.
+ */
+OrderBackup::OrderBackup()
+{
 }
```

Comment:
```suggestion
OrderBackup::OrderBackup() = default;
```
Or maybe just `= default` in the class definition?